conformance/requirements.txt

```diff
@@ -0,0 +1,7 @@
+tomli
```

Comment:
I read earlier that we require 3.12, so shouldn't we be able to use the stdlib's tomllib module instead?